lib/media/segment_prefetch.js

```diff
@@ -55,28 +73,40 @@ shaka.media.SegmentPrefetch = class {
    * @public
    */
   prefetchSegments(startReference, skipFirst = false) {
```

Comment:
I know it's a change, but since we now have prefetchSegmentsByTime, we should remove this method and have everything use prefetchSegmentsByTime. We should also remove the fetchInit parameter and make it always true. This change involves making the prefetchInitSegment method private.